src/utils/utils.ts

```diff
@@ -61,3 +61,60 @@ export function settingsListToTree(list: Setting[]): Tree {
   });
   return root;
 }
+
+
+export function deleteLocalKey(key : string, id : string) {
+  var toUpdate = JSON.parse(localStorage.getItem(key));
+  delete toUpdate[id];
+  localStorage.setItem('api_functions', JSON.stringify(toUpdate));
```

Comment:
i think you can move this function to the component because it is specifically for the functions.

Reply:
Just a mistake. Should be localStorage.setItem(key, JSON.stringify(toUpdate));